wcfsetup/install/files/lib/system/message/embedded/object/MessageEmbeddedObjectManager.class.php

```diff
@@ -66,6 +66,12 @@ class MessageEmbeddedObjectManager extends SingletonFactory {
 		'insert' => [],
 		'remove' => []
 	];
+
+	/**
+	 * A list of previous active message settings used to restore
+	 * the internal state in case of nested message processing.
+	 */
+	protected $acitveMessageHistory = [];
```

Comment:
```suggestion
	protected $activeMessageHistory = [];
```
(adjust the other location yourself, please).